### PR TITLE
Deprecate news sources

### PR DESCRIPTION
**Pull request by drewvolz**
- Paired with https://github.com/StoDevX/AAO-React-Native/pull/5363 to handle clients that still request these endpoints.
- Discussion for deprecated endpoints https://github.com/frog-pond/ccc-server/discussions/564  

<details>
<summary>📸 Screenshots</summary>

KSTO | PoliticOle
--|--
 | 

</details>

### CODE REVIEW

File: modules/node_modules/@frogpond/ccc-wpjson-feed/index.js [hunk 1]
```diff
@@ -53,3 +53,11 @@ export function convertWpJsonItemToStory(item) {
 		title: JSDOM.fragment(item.title.rendered).textContent.trim(),
 	}
 }
+
+export function deprecatedWpJson() {
+	return [{
+		content: "",
+		excerpt: "Please update your app.",
+		title: "Deprecated endpoint",
```
Comment (hawkrives) [inline modules/node_modules/@frogpond/ccc-wpjson-feed/index.js:61]: "This news source is no longer being updated." maybe?

File: modules/node_modules/@frogpond/ccc-wpjson-feed/index.js [hunk 2]
```diff
@@ -53,3 +53,11 @@ export function convertWpJsonItemToStory(item) {
 		title: JSDOM.fragment(item.title.rendered).textContent.trim(),
 	}
 }
+
+export function deprecatedWpJson() {
+	return [{
+		content: "",
```
Comment (hawkrives) [inline modules/node_modules/@frogpond/ccc-wpjson-feed/index.js:59]: Should we add a link to the App Store entry / GitHub page… maybe a GitHub issue where we describe the message?

Comment (rye) [reply modules/node_modules/@frogpond/ccc-wpjson-feed/index.js:59]: I like the idea of a GitHub issue link!

File: modules/node_modules/@frogpond/ccc-wpjson-feed/index.js (hunk 1 above)
Comment (drewvolz) [reply modules/node_modules/@frogpond/ccc-wpjson-feed/index.js:61]: That is an improvement. If we provide a link to learn more, do our chevrons indicate they can learn more by tapping the row, or should we explicitly provide that in the excerpt?

> Please update your app
> This news source is no longer being updated. Tap to learn more about why you're seeing this message.

Comment (hawkrives) [reply modules/node_modules/@frogpond/ccc-wpjson-feed/index.js:61]: I don't think we need to say "tap to learn more", imo.

File: modules/node_modules/@frogpond/ccc-wpjson-feed/index.js (hunk 2 above)
Comment (hawkrives) [reply modules/node_modules/@frogpond/ccc-wpjson-feed/index.js:59]: Server repo, I think

**Review by rye — APPROVED** [review]

**Comment by rye** [comment]
We _do_ have Discussions... could the linked issue be turned into one?

**Review by rye — APPROVED** [review]

**Review by hawkrives — APPROVED** [review]

**Pull request merged**